src/moves/FreeScroll.ts

```diff
@@ -56,12 +58,12 @@ class FreeScroll extends Snap {
   }
 
   public findPanelWhenInterrupted(ctx: MoveTypeContext): DestinationInfo {
-    const { viewport } = ctx;
+    const { viewport, axesEvent } = ctx;
 
     return {
       panel: viewport.getNearestPanel()!,
       destPos: viewport.getCameraPosition(),
-      duration: viewport.options.duration,
+      duration: Math.max(axesEvent.duration, viewport.options.duration),
```

Comment:
You've committed with wrong file included.
Please do reset this file.

Reply:
@WoodNeck 
The preact PR is likely to be postponed for now. 

I will fix events first by looking at my hands and then fix freeScroll issue commit contents soon.